src/componenets/Header/productbutton/productbutton.js

```diff
@@ -0,0 +1,5 @@
+function productAddButton({ disabled, children }) {
```

Comment:
JSX를 반환하는 리액트컴포넌트니까 파스칼케이스인편이 좋겠습니다
```suggestion
function ProductAddButton({ disabled, children }) {
```